src/auth/auth.js

```diff
@@ -121,15 +76,13 @@ Auth.prototype.setUser = function(user) {
 	this._user = user
 }
 
-Auth.prototype.login = function(token) {
-	this.setUser(UserFromToken(token))
-
+Auth.prototype.login = async function() {
+	this.loading.state = true
+	await this.resumeSession(false) // fetch user session data
```

Comment:
Is this correct? The login doesn't seem to log in?

Reply:
Auth.prototype.login is now in Auth.prototype.loadSession to make it clearer what it does.